app/src/device/kotlin/de/digitalService/useID/ui/coordinators/IdentificationCoordinator.kt

```diff
@@ -56,7 +56,13 @@ class IdentificationCoordinator @Inject constructor(
     private var reachedScanState = false
 
     fun startIdentificationProcess(tcTokenURL: String) {
+        logger.debug("Start identification process.")
         idCardManager.cancelTask()
+        reachedScanState = false
+        CoroutineScope(coroutineContextProvider.IO).launch {
+            _fetchMetadataEventFlow.emit(FetchMetadataEvent.Started)
```

Comment:
`startIdentification()` already emits this event, is this a required duplication?

Reply:
No, you're right. It's not. 👍